presto-main/src/main/java/io/prestosql/event/QueryMonitor.java

```diff
@@ -339,6 +371,58 @@ private static QueryIOMetadata getQueryIOMetadata(QueryInfo queryInfo)
         return new QueryIOMetadata(inputs.build(), output);
     }
 
+    private static Multimap<QualifiedObjectName, OperatorStats> extractTableScanOperatorStats(QueryInfo queryInfo)
```

Comment:
optional suggestion: the name may sound like the method is trying to get stats for `TableScanOperator`, which might not be true. May be switching the order can help e.g. `extractStatsForTableScan`?